python/tidy/servo_tidy/tidy.py

```diff
@@ -841,6 +855,22 @@ def check_config_file(config_file, print_text=True):
             current_table = table_name
             continue
 
+        # Print invalid listed ignored directories
+        if current_table == "ignore" and invalid_dirs:
+            for d in invalid_dirs:
+                if any(fd in line for fd in ["'{}'".format(d), '"{}"'.format(d)]):
+                    yield config_file, idx + 1, "invalid listed ignored directory '%s'" % d
```

Comment:
Oh, and also let's have a more specific help message, something like, `"ignored directory %r doesn't exist" % d`